CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -75,6 +76,17 @@ public static IList<string> GetHostsFromEnvironment(ILogger logger, string envVa
             return hosts;
         }
 
+        public static bool? ForceCanShowDialogTo()
+        {
+            var fromEnv = Environment.GetEnvironmentVariable(ForceCanShowDialogEnvVar);
+            if(string.IsNullOrWhiteSpace(fromEnv) || !bool.TryParse(fromEnv, out var parsed))
+            {
+                return default;
```

Comment:
Just checked the default value of a nullable bool. I think it's null which is what we want here but sometimes I can't understand English. 

"An instance for which the HasValue property is false and the Value property is undefined. That default value is also known as the null value of a nullable value type."
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values

Reply:
Yes. `default` where an expression of type `Nullable<bool>` is expected is equivalent to `null` in the same place, which are both syntactic sugar for `default(Nullable<bool>)`. The reason for the strange wording in the doc is that `Nullable<T>` is still a value type, so it technically can't be null per se, but syntactic sugar in the language sort of hides that.